src/scales/scale.linear.js

```diff
@@ -11,123 +11,129 @@ var defaultConfig = {
 	}
 };
 
+var DEFAULT_MIN = 0;
+var DEFAULT_MAX = 1;
+
+function getMatchingVisibleMetas(scale) {
+	var isHorizontal = scale.isHorizontal();
+
+	return scale.chart._getSortedVisibleDatasetMetas()
+		.filter(function(meta) {
+			return isHorizontal ? meta.xAxisID === scale.id : meta.yAxisID === scale.id;
+		});
+}
+
+function getStack(stacks, stacked, meta) {
+	var key = [
+		meta.type,
+		// we have a separate stack for stack=undefined datasets when the opts.stacked is undefined
+		((stacked === undefined && meta.stack === undefined) ? meta.index : ''),
```

Comment:
we probably don't need the parentheses on this line

Reply:
Parentheses removed